src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -105,35 +107,54 @@ private String extractSurnameFromOcrData(Envelope envelope) {
             return null;
         }
 
-        String surnameOcrFieldName = serviceConfigProvider.getConfig(envelope.container)
-            .getSurnameOcrFieldName(envelope.formType);
 
-        List<String> surnameList = envelope.ocrData
-            .stream()
-            .filter(ocrData -> ocrData.name.equals(surnameOcrFieldName))
-            .map(ocrData -> ocrData.value)
-            .collect(toList());
-        if (surnameList.size() == 0) {
-            LOGGER.info(
-                "Surname not found in OCR data. "
-                    + "Surname Ocr Field Name: {}. Envelope id: {}, Case Ref: {}, Jurisdiction: {}",
-                surnameOcrFieldName,
-                envelope.id,
-                envelope.caseRef,
-                envelope.jurisdiction
-            );
+        Optional<List<String>> surnameOcrFieldNameListOp =
+            serviceConfigProvider
+                .getConfig(envelope.container)
+                .getSurnameOcrFieldNameList(envelope.formType);
+
+
+        if (!surnameOcrFieldNameListOp.isPresent()) {
```

Comment:
Do we need to log the message when surname field is not found?

Reply:
it may create too much logs. I am not sure we need.